### PR TITLE
Cleanup of README + alphabetical order of crates

### PR DESCRIPTION
**Pull request by phip1611**
### Summary of the PR

At my company Cyberus Technology, we think about upstreaming certain functionality. As preparation and to get familiar with the code base and the contribution process, here is a first PR. It synchronizes the README with the crates of this repository and orders all crates alphabetically.

btw: I noticed that some things seem to be unmaintained or unused. For example:
- `virtio-device` is not on crates.io 
- `virtio-devices` is on crates.io but its GitHub repository doesn't exist
- `virtio-console` is not on crates.io

Any comments on those?

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR are signed (with `git commit -s`), and the commit
  message has max 60 characters for the summary and max 75 characters for each
  description line.
- [ ] ~All added/changed functionality has a corresponding unit/integration
  test.~
- [ ] ~Any newly added `unsafe` code is properly documented.~

### CODE REVIEW

**Comment by lauralt** [comment]
Hi! Thanks for the contribution.

> I noticed that some things seem to be unmaintained or unused. For example:
> 
>     * `virtio-device` is not on crates.io
> 

I was thinking that we could publish the crate in the near future, but before that there are a few improvements needed to documentation (like code examples), also some more tests needed. I will open a separate issue for this one.

>     * `virtio-devices` is on crates.io but its GitHub repository doesn't exist

Hmm,  I think this is something different than the `virtio-devices` from this workspace. @andreeaflorescu might have more info about this.

> 
>     * `virtio-console` is not on crates.io
> 

It was developed recently as a bachelor project, didn't consider yet when we could/should publish it.

File: README.md [hunk 1]
```diff
@@ -5,13 +5,16 @@ The `vm-virtio` workspace provides
 abstractions and implementation for the virtio queue and devices. For now, it
 consists of the following crates:
 
+- `virtio-bindings` -> provides Rust FFI bindings to the corresponding Linux kernel API;
```
Comment (lauralt) [inline README.md:8]: nit: we usually limit lines in md files to 80 chars.

**Review by lauralt — DISMISSED** [review]
You need to also sign off your commit (`git commit -s --amend`).

**Comment by andreeaflorescu** [comment]
I don't really know what virtio-devices is. I suspect @sameo published an empty crate to reserve the name. We are not going to publish a virtio-devices crate anytime soon I think because we want to have devices published as part of individual crates (most likely one per devices). As @lauralt already pointed out, we intend to publish virtio-device.

**Comment by phip1611** [comment]
> You need to also sign off your commit (`git commit -s --amend`).

@lauralt My commit is signed/verified? Am I missing something? 🤔 
image

File: README.md (hunk 1 above)
Comment (phip1611) [reply README.md]: fixed

**Comment by lauralt** [comment]
> > You need to also sign off your commit (`git commit -s --amend`).
> 
> @lauralt My commit is signed/verified? Am I missing something? thinking
> 
> image

That is the GPG signature, you also need to add the `Signed-off-by` line to your commit message. The difference is explained [here](https://stackoverflow.com/questions/64840485/what-is-the-difference-between-git-commit-s-and-s).

**Comment by phip1611** [comment]
@lauralt oh, my bad. Thanks for the hint!

**Review by andreeaflorescu — APPROVED** [review]

**Review by lauralt — APPROVED** [review]

**Pull request merged**